[V3 Audio] Fix for playlist queue duplicates

aikaterna opened the pull request:
### Type

- [x] Bugfix
- [ ] Enhancement
- [ ] New feature

### Description of the changes
Fix for #1888 and playlist names are limited to one word.

aikaterna committed:
[V3 Audio] Fix for playlist queue duplicates

And some sanitizing of playlist names.

Crossedfall commented on redbot/cogs/audio/audio.py:
```diff
@@ -794,11 +795,13 @@ def check(r, u):
                 playlist_name_msg = await ctx.bot.wait_for("message", timeout=15.0, check=check)
-                playlist_name = str(playlist_name_msg.content)
+                playlist_name = re.sub(
+                    "[^a-zA-Z0-9]", "", str(playlist_name_msg.content.split(" ")[0])
```
This will terminate the line at the first space even when contained in double quotes. This doesn't match up with how you can have a playlist name containing spaces if you don't utilize the `wait` feature.

e.g.

`[p]playlist queue "Test Playlist"`
Saves a playlist called, "Test Playlist"

`[p]playlist queue`
_prompt for the user to enter in a name_
`Test Playlist` or `"Test Playlist"`
New playlist called, "Test"

Tobotimus replied:
~~Why don't we get rid of the double quotes rule and make `playlist_name` consume rest?~~ Never mind me, that's not really related to your point

Tobotimus committed:
Merge branch 'V3/develop' into patch-5

Tobotimus committed:
Merge branch 'V3/develop' into patch-5

Tobotimus committed:
Merge branch 'V3/develop' into patch-5

aikaterna committed:
[V3 Audio] Playlist naming standardization

Enforced single-word playlist name across all playlist commands, removed A-Z 0-9 name standardization. [p]playlist delete will still accept playlist names with quotes as there should be a way to remove already-existing playlists with spaces in their name.

aikaterna committed:
[V3 Audio] Black formatting

aikaterna committed:
Merge branch 'V3/develop' into patch-5

Kowlin committed:
Merge branch 'V3/develop' into patch-5
